core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerUtils.scala

```diff
@@ -283,11 +283,11 @@ private[mesos] trait MesosSchedulerUtils extends Logging {
    *  are separated by ':'. The ':' implies equality (for singular values) and "is one of" for
    *  multiple values (comma separated). For example:
    *  {{{
-   *  parseConstraintString("tachyon:true;zone:us-east-1a,us-east-1b")
+   *  parseConstraintString("os:centos7;zone:us-east-1a,us-east-1b")
```

Comment:
The key-value `tachyon:true` has been changed to `os:centos7`, since `os` is an example constrain which appears in [Mesos official docs](http://mesos.apache.org/documentation/attributes-resources/).

Also note that the `true` as in `tachyon:true` here is actually treated as normal texts, so changing it to `os:centos7` doesn't make any difference.